src/components/User/UserComponent.jsx

```diff
@@ -33,25 +33,27 @@ export default class UserComponent extends React.Component {
   }
 
   handleRowChange = (e, data) => {
+    const { activePage, selected } = this.props;
     this.setState({
       limit: data.value,
       users: []
     });
     this.props.resetUsers();
-    this.retrieveUsers(this.props.activePage, data.value);
+    this.retrieveUsers(activePage, data.value, selected);
   };
 
   handlePaginationChange = (e, { activePage }) => {
     this.props.setActivePage(activePage);
     const currentPageList = this.props.users[`page_${activePage}`];
     if (isEmpty(currentPageList)) {
-      this.retrieveUsers(activePage, this.state.limit);
+      this.retrieveUsers(activePage, this.state.limit, this.props.selected);
     }
   };
 
-  retrieveUsers = (activePage, limit) => {
+  retrieveUsers = (activePage, limit, filters) => {
     if (checkIfCutoffExceeded(activePage, limit)) {
-      const url = `users?page=${activePage}&page_size=${limit}`;
+      const url = `users?page=${activePage}&page_size=${limit}&asset_count=${filters[
```

Comment:
Please move the [constructUrl](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/_actions/users.actions.js#L19) to a helper file then use it here as:
```jsx
const url = constructUr (activePage, limit, filters);
```
Don't forget to import the `constructUrl` in [users.actions.js](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/_actions/users.actions.js) as it is being used in that file as well